### PR TITLE
ACS-3352 GET e2es for newly mapped other fields

### PR DESCRIPTION
**Pull request by KikoUK**

### CODE REVIEW

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java [hunk 1]
```diff
@@ -48,9 +48,9 @@
      *
      * @return The created rule model.
      */
-    public static RestRuleModel createRuleModelWithDefaultValues()
+    public static RestRuleModel createRuleModelWithModifiedValues()
```
Comment (krdabrowski) [inline packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java:51]: I'm not getting the name change, why it contains "ModifiedValues" when the method creates a rule with predefined default values?

Comment (KikoUK) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java:51]: The idea is that if you create a rule without specifying any of the new "other" values, they all have some value. 
The method name above was confusing (at least to me) cos it modifies the default values of the fields, but it was called `defaultValues`.

Comment (krdabrowski) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java:51]: Ohh ok, now I understand - I'm ok with that.

**Review by krdabrowski — APPROVED** [review]

File: packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java [hunk 2]
```diff
@@ -95,8 +95,8 @@ public static RestRuleModel createRuleModel(String name, List<RestActionBodyExec
     public static RestActionBodyExecTemplateModel createDefaultActionModel()
     {
         RestActionBodyExecTemplateModel restActionModel = new RestActionBodyExecTemplateModel();
-        restActionModel.setActionDefinitionId("add-features");
-        restActionModel.setParams(Map.of("aspect-name", "cm:audio"));
+        restActionModel.setActionDefinitionId("set-property-value");
+        restActionModel.setParams(Map.of("aspect-name", "cm:audio", "actionContext", "rule"));
```
Comment (mpichura) [inline packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java:99]: Entry of `"actionContext" -> "rule"` should not be in the action parameters when creating new action. 
This entry is added automatically while validating rule actions when creating/updating rule (see [ACS-3335](https://alfresco.atlassian.net/browse/ACS-3335) and [ACS-3336](https://alfresco.atlassian.net/browse/ACS-3336))
If you are using this method to prepare input model for newly created or updated rule with new action - you should not  add `"actionContext" -> "rule"` entry in action parameters.
There code with a method adding that entry to each action (`RulesTestsUtils::addActionContextParams`) should already be merged. Mentioned method can be used to enhance rule action when creating expected rule object.

Comment (KikoUK) [reply packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/RulesTestsUtils.java]: I saw your changes are already merged and modified the actions, thanks 🙏

**Comment by mpichura** [comment]
Failing rule related tests :(

**Comment by KikoUK** [comment]
> Failing rule related tests :(

Just pushed a fix

**Review by mpichura — APPROVED** [review]

**Pull request merged**